tests/test_page.py

```diff
@@ -1203,8 +1203,11 @@ def test_merge_transformed_page_into_blank():
     inserted_blank = writer.add_page(blank)
     assert blank.page_number is None  # the inserted page is a clone
     assert inserted_blank.page_number == len(writer.pages) - 1
-    del writer._pages.get_object()["/Kids"][-1]
+    writer.remove_page(inserted_blank.indirect_reference)
```

Comment:
As this has been an internal attribute (`_pages`), we do not consider this a breaking change?

Reply:
this hack for test coverage has been rewritten. remove_page is a slight extension of the existing 'del writer.pages[int]` allowing to pass a `PageObject` or an `IndirectObject`